persistence/src/main/scala/akka/persistence/kafka/journal/KafkaJournalConfig.scala

```diff
@@ -56,6 +60,7 @@ final case class KafkaJournalConfig(
   callTimeThresholds: CallTimeThresholds = CallTimeThresholds.default,
   jsonCodec: KafkaJournalConfig.JsonCodec = KafkaJournalConfig.JsonCodec.Default,
   consumerPool: ConsumerPoolConfig = ConsumerPoolConfig.Default,
+  dataIntegrity: DataIntegrityConfig = DataIntegrityConfig.Default,
```

Comment:
seems that MiMa check wants us to add custom constructor `apply` without `dataIntegrity` - probably we can do that in companion object...

Reply:
no, I disabled MiMa for now. Will bring it back after merging the PR